conn_windows.go

```diff
@@ -7,3 +7,7 @@ import "net"
 func newConn(c net.PacketConn) (connection, error) {
 	return &basicConn{PacketConn: c}, nil
 }
+
+func inspectReadBuffer(net.PacketConn) (int, error) {
+	return 0, nil
```

Comment:
Any particular reason why golang.org/x/sys wasn't used? `GetsockoptInt` should work for Windows in that package.

Reply:
@dswarbrick Not sure I understand. Can you elaborate?

Reply:
It appears that the `inspectReadBuffer` function here is a no-op on Windows, presumably because `syscall.GetsockoptInt` in the standard library is not implemented for Windows:
```go
func GetsockoptInt(fd Handle, level, opt int) (int, error) { return -1, EWINDOWS }
```
Since the syscall package is frozen, that functionality is not likely to make it into the standard library. However, it _is_ implemented in the golang.org/x/sys/windows package, as described in the link I posted.

Reply:
@dswarbrick Thanks for pointing this out, I wasn't aware of the syscall package being frozen.
Would you mind sending us a PR that adds this functionality on Windows?